### PR TITLE
curve: Bump version to 4.1.3

### PR DESCRIPTION
**Pull request by rozbb**

### CODE REVIEW

**Comment by tarcieri** [comment]
Would be good to get #661 in this release too

File: curve25519-dalek/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ major series.
 
 ## 4.x series
 
+### 4.1.3
+
+* Security: Fix timing leak in Scalar subtraction on u32, u64, fiat_u32, and fiat_u64 backends
```
Comment (tarcieri) [inline curve25519-dalek/CHANGELOG.md:10]: Should we also include various nightly/lint-related fixes like #625, #636, #638, #639, and #652?

Comment (rozbb) [reply curve25519-dalek/CHANGELOG.md:10]: Yup, doing now

Comment (rozbb) [reply curve25519-dalek/CHANGELOG.md:10]: Check again

Comment (tarcieri) [reply curve25519-dalek/CHANGELOG.md:10]: Looks good (aside from a rebase on #661)

**Review by tarcieri — APPROVED** [review]

**Pull request merged**